expression/builtin_time.go

```diff
@@ -1762,69 +1766,75 @@ type builtinConvertTzSig struct {
 func (b *builtinConvertTzSig) eval(row []types.Datum) (d types.Datum, err error) {
 	args, err := b.evalArgs(row)
 	if err != nil {
-		return d, errors.Trace(err)
+		return
```

Comment:
if err != nil, please use return d, errors.Trace(err) for tracing the error in the log.
the same as line 1784, 1803, 1808, 1811